pricing.js

```diff
@@ -55,20 +55,40 @@ function calculateLTDPrice(product, employee, selectedOptions) {
   return price
 }
 
+function calculateCommuterCost(product, selectedOptions) {
+  let price = 0
+
+  if (selectedOptions.benefits === 'parking') {
+
+    return price += 250
+
+  }
+  else {
+    price += 84.75
+
+    return price
+  }
+}
```

Comment:
A couple of problems here. One, you aren't using the data from the product so if there was a commuter product for a company where parking costs $300 you'd calculate the wrong price for them. The second is this isn't very futureproof. If we add a bicycle type to commuter this could wouldn't handle that. It is possible to write this function in a way that would handle that without a change.